src/index.js

```diff
@@ -37,248 +34,105 @@ export default function supernova(env) {
       data,
     },
     component: () => {
-      const [objectData, setObjectData] = useState(null);
-      const [styling, setStyling] = useState(null);
-      const [expandedState, setExpandedState] = useState(null);
-      const [linked, setLinked] = useState(false);
-      const [selectionState, setSelectionState] = useState([]);
-      const [transform, setTransform] = useState(null);
-      const [initialZoom, setInitialZoom] = useState(null);
       const layout = useStaleLayout();
       const model = useModel();
       const element = useElement();
       const Theme = useTheme();
       const options = useOptions();
-      const [opts] = useState(options);
       const rect = useRect();
-      const selectionsAPI = useSelections();
       const constraints = useConstraints();
-      const [selectionsAndTransform] = useState({
-        api: selectionsAPI,
-        setState: setSelectionState,
-        linked: false,
+      const translator = useTranslator();
+      const [preRenderData, setPreRenderData] = useState(null);
+      const [styling, setStyling] = useState(null);
+      const [expandedState, setExpandedState] = useState(null);
+      const [transform, setTransform] = useState(null);
+      const [initialZoom, setInitialZoom] = useState(null);
+      const [storageState] = useState({
```

Comment:
What is the storageState?

Reply:
`selectionsAndTransorm` minus the selection stuff. So constraints and transform at this point. Wanted to call it something with state, since it is a state. Name suggestions are very welcome 🙃